README.md

```diff
@@ -203,7 +203,12 @@ Available targets:
 
 ```
 
+## Extending `build-harness` with targets from another repo
```

Comment:
Please update `README.yaml` instead, then rerun `make readme`

Reply:
Thanks for the nudge in the right direction for documentation updates.  Sorry for the merge noise!